website/docs/r/network_routed_v2.html.markdown

```diff
@@ -154,8 +154,11 @@ The following arguments are supported:
   from `edge_gateway_id`)
 * `name` - (Required) A unique name for the network
 * `description` - (Optional) An optional description of the network
-* `interface_type` - (Optional) An interface for the network. One of `internal` (default), `subinterface`, 
-  `distributed` (requires the edge gateway to support distributed networks). NSX-T supports only `internal`
+* `interface_type` - (Optional) An interface for the network. One of `internal` (default),
+  `subinterface`, `distributed`, `non_distributed` (requires the edge gateway to support distributed
+  networks). NSX-T supports only `internal` and `non_distributed` (*v3.14+*, requires Edge Gateway
+  to have [non distributed routing
```

Comment:
Would be easier to read and be less error prone with a dash: "non-distributed", like in VCD docs:
https://docs.vmware.com/en/VMware-Cloud-Director/10.5/VMware-Cloud-Director-Service-Provider-Admin-Guide/GUID-16EA3560-7927-4990-AB19-068672410986.html

```suggestion
  to have [non-distributed routing
```

Maybe there are more places to tune like this too?

Reply:
Changed. Global search doesn't find more.